slack-notify/action.yml

```diff
@@ -0,0 +1,41 @@
+name: Slack Notification
+
+inputs:
+  slack-notify-override:
```

Comment:
I wonder if "send_all_notifications" or "send_all" would be clearer as to what's going on?

Reply:
I like `send_all_notifications`, but yea that helps with context